DigitalLearningSolutions.Web/Views/RegisterDelegateByCentre/WelcomeEmail.cshtml

```diff
@@ -25,29 +24,13 @@
     <h1 class="nhsuk-heading-xl">Welcome email</h1>
 
     <form class="nhsuk-u-margin-bottom-3" method="post" asp-action="WelcomeEmail">
-      <div class="nhsuk-checkboxes nhsuk-checkboxes--conditional nhsuk-u-margin-bottom-4">
-        <div class="nhsuk-checkboxes__item">
-          <input class="nhsuk-checkboxes__input"
-                 id="ShouldSendEmail"
-                 name="ShouldSendEmail"
-                 asp-for="ShouldSendEmail"
-                 type="checkbox"
-                 value="true"
-                 aria-controls="@dateInputId"
-                 aria-expanded="@(Model.ShouldSendEmail ? "true" : "false")">
-          <label class="nhsuk-label nhsuk-checkboxes__label" for="ShouldSendEmail">
-            Send welcome email to registered delegate
-          </label>
-        </div>
-
-        <vc:date-input id="@dateInputId"
-                       label="Deliver email on or after"
-                       day-id="Day"
-                       month-id="Month"
-                       year-id="Year"
-                       css-class="@emailDateCss"
-                       hint-text-lines="@hintTextLines" />
-      </div>
+      <vc:date-input id="@dateInputId"
+                     label="Send welcome email to registered delegate, to be delivered on or after"
+                     day-id="Day"
+                     month-id="Month"
+                     year-id="Year"
+                     css-class="@(errorHasOccurred ? " nhsuk-u-padding-left-5" : "")"
+                     hint-text-lines="@hintTextLines" />

```

Comment:
Also noticed that this date input looks a little funky when there's an error, with the red bar touching the button below
![image](https://user-images.githubusercontent.com/70278044/178278816-4bab4227-509f-4e59-ac82-064e98573ef1.png)